api/app/signals/apps/email_integrations/utils.py

```diff
@@ -25,11 +25,11 @@
 from signals.apps.signals.tests.valid_locations import STADHUIS
 
 
-def create_feedback_and_mail_context(signal: Signal):
+def create_feedback_and_mail_context(signal: Signal) -> dict:
     """
     Util functions to create the feedback object and create the context needed for the mail
     """
-    # Note: While the questionnaires app support for feedback requests is under
+    # Note: While the questionnaire's app support for feedback requests is under
```

Comment:
`questionnaires app` is a name here, so the apostrophe is inappropriate here (a capital Q could be considered).